### ISSUE TITLE
request

### ISSUE THREAD

**Issue by IF-tiger** [Luwx/Lightly#199]
my device is arch and wayland
![image](https://user-images.githubusercontent.com/60809013/204170659-240f099c-19e7-4832-b54c-69ca7cf28b39.png)
![image](https://user-images.githubusercontent.com/60809013/204170929-cd95259a-78b4-4867-a36e-55d1dc4f45cc.png)

description：
As you move the mouse through each item, the residual background color does not disappear immediately, but leaves a lighter color and does not disappear

end： This is a great project, and if you still have spare time to continue fixing, thank you for your work

**Comment by IF-tiger** [comment]
This bug may have something to do with Wayland itself

**Comment by Etaash-mathamsetty** [comment]
> This bug may have something to do with Wayland itself

this is a wayland bug in kde, I have the same issue with breeze

**Comment by luisbocanegra** [comment]
Try setting Lightly > Transparency > Menu to full opaque. Does it go away?

**Comment by Etaash-mathamsetty** [comment]
> Try setting Lightly > Transparency > Menu to full opaque. Does it go away?

yes it does, but I don't think I want to do that permanently lol

I want to enjoy this theme before it finally completely dies in plasma 6

**Comment by boehs** [comment]
> before it finally completely dies in plasma 6

It won't
